### DISCUSSION TITLE
Version 2.1.0

### DISCUSSION THREAD

**Discussion by patkon** [flatCore/flatCore-CMS#73] [General]
**Please note:** We have dropped PHP 5.X support. Minimal requirement is PHP 7.3+ now.

## What's Changed
* Develop by @patkon in https://github.com/flatCore/flatCore-CMS/pull/63
* Merge Develop Branch by @patkon in https://github.com/flatCore/flatCore-CMS/pull/72

**Full Changelog**: https://github.com/flatCore/flatCore-CMS/compare/v2.0.9...v2.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/flatCore/flatCore-CMS/releases/tag/v2.1.0'>Version 2.1.0</a>.</em>

**Comment by Madtricksystem** [comment]
Ich teste gerade das main Branch ohne weitere Module/Plugins und möchte Feedback zur Info geben. Bisher ist mir folgendes aufgefallen:
1. Im Backend lassen sich Benutzern keine Avatare hinterlegen. Bug oder mir entgeht wie? ;-)
2. Beiträge (Nachricht), die ohne Bildauswahl erstellt werden, haben in der aufgerufenen Webseite ein "Bild nicht verfügbar" Symbol.
3. Beiträge (Galerie): Nach dem Speichern und nach dem gefolgten Bilder-Upload kommt direkt eine weiße Seite mit Fehlermeldung: Error: CSRF Token is empty

**Reply by patkon** [reply]
Hallo Dirk,
danke für Dein Feedback.
**Zu Punkt 1.:** Das ist/war Absicht. Als es noch keine Benutzerrollen im Backend gab wollte ich verhindern, dass man irgendwelche Benutzerbilder hochladen kann. Ist vielleicht an der Zeit, diese Beschränkung aufzuheben.

**Zu Punkt 2:** Wenn Du in den Einstellungen unter System > Beiträge > Standard verwenden auswählst, wird das Bild angezeigt welches Du unter System > Bilder/Uploads > Thumbnail ausgewählt hast. Hast Du hier keines ausgewählt, entsteht der Fehler. Wenn Du unter System > Beiträge > Standard Bild die Option "Kein Bild verwenden" auswählst, erscheint auch kein Bild. Klingt erstmal kompliziert, ist es aber eigentlich nicht, ich kann es nur schlecht erklären.

**Zu Punkt 3:** Der Fehler ist mit dem nächsten Update behoben (im Develop Branch ist er es schon).
